Makefile.am

```diff
@@ -185,6 +185,7 @@ TESTS+=test/test-cases/regression/request-body-parser-xml-validade-dtd.json
 TESTS+=test/test-cases/regression/rule-920120.json
 TESTS+=test/test-cases/regression/rule-920200.json
 TESTS+=test/test-cases/regression/rule-920274.json
+TESTS+=test/test-cases/regression/rule-920450.json
```

Comment:
I would expect a different name for that file. Explained below.

Reply:
Fixed.